README.md

```diff
@@ -20,8 +20,14 @@ If you'd like to help out, please see [CONTRIBUTING.md](CONTRIBUTING.md).
 
 ## Release history
 
-* Next release:
+* Release 0.21.2 (2023-06-14)
   - Bump MSRV to 1.60 to track similar change in dependencies.
+  - Differentiate between unexpected and expected EOF in `Stream` and `OwnedStream`.
+  - `RootCertStore::add_parsable_certificates` now takes a `&[impl AsRef<[u8]>]`.
+  - Add QUIC V2 support.
+  - Upgrade `env_logger` to v0.10.
+  - Correctly provide an empty session ID if none is available for TLS 1.2.
+  - Bug fix: abort clients offering "pre_shared_key" without a "psk_key_exchange_modes" extension.
```

Comment:
Historically we haven't really made node of minor bugfixes like this in the changelog (I would probably only keep the QUIC V2 support and the `Stream` EOF handling), @ctz do you feel we should change that?

Reply:
Yeah agree that it's a minor bug fix (and never going to be seen in the wild). I probably wouldn't put it in a change log (cos it's not interesting or actionable for users).